scripts/esbuild/dev-server.ts

```diff
@@ -0,0 +1,304 @@
+import { builtinModules } from 'node:module';
+import { join } from 'node:path';
+
+import type { BuildOptions as ESBuildOptions, Plugin } from 'esbuild';
+import { replace } from 'esbuild-plugin-replace';
+import fs from 'fs-extra';
+import ts from 'typescript';
+
+import { createContentTypeData } from '../bundles/plugins/content-types-plugin';
+import { bundleExternal, sysNodeBundleCacheDir } from '../bundles/sys-node';
+import { getBanner } from '../utils/banner';
+import { type BuildOptions, createReplaceData } from '../utils/options';
+import { writePkgJson } from '../utils/write-pkg-json';
+import { getBaseEsbuildOptions, getEsbuildAliases, runBuilds } from './util';
+const connectorName = 'connector.html';
```

Comment:
style nit: usually for constants I'll go for `const VARIABLE_NAME = ...` type of style